tox.ini

```diff
@@ -9,6 +9,7 @@ deps =
     pytest-mock
```

Comment:
GitHub won't let me comment directly, but we can probably remove the `passenv` line above that was there to pass through credentials for the integration tests.